draft-ietf-quic-recovery.md

```diff
@@ -201,6 +201,21 @@ implemented in QUIC.
 
 ### Fast Retransmit
 
+Fast retransmit can be run two different modes: packet number based and time
+based.
+
+We derive this default from recommendations for TCP loss recovery {{?RFC5681}}
+{{?RFC6675}}. It is possible for networks to exhibit higher degrees of
+reordering, causing a sender to detect spurious losses. Making QUIC deal with
+such networks is important open research, and implementers are encouraged to
+explore this space. Detecting spurious losses leads to unnecessary
+retransmissions and may result in degraded performance due to the actions of
+the congestion controller upon detecting loss. Implementers MAY use algorithms
+developed for TCP, such as TCP-NCR {{?RFC4653}}, to improve QUIC's reordering
+resilience, though care should be taken to map TCP specifics to QUIC correctly.
```

Comment:
I think it's worth commenting that both modes could be run simultaneously, but the psuedocode only describes how to run one at once.